src/__tests__/useWindowSize.test.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import useWindowSize from "../useWindowSize"
+import { renderHook, cleanup, act } from 'react-hooks-testing-library'
```

Comment:
`cleanup` not used

Reply:
Added cleanup according to the example on their readme